### DISCUSSION TITLE
Why "Creating initial user..." at each start despite a user already exists ?

### DISCUSSION THREAD

**Discussion by KaKi87** [tolgee/tolgee-platform#1025] [Q&A] [answered]
> `INFO 19879 --- [           main] io.tolgee.InitialUserCreatorRunner       : Creating initial user...`

Thanks

**Comment by JanCizmar** [comment, marked as answer]
It's just logging before it checks whether the user exists.

**Comment by JanCizmar** [comment]
Here is a PR:
https://github.com/tolgee/tolgee-platform/pull/1032
